packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/general-information.html

```diff
@@ -121,33 +123,42 @@
                                                       'oui-badge_warning': $ctrl.notebook.stateInfo.stateGroup.warning,
                                                       'oui-badge_error': $ctrl.notebook.stateInfo.stateGroup.error}"
                         >
-                                                    <span
-                                                        data-ng-bind="'pci_notebooks_general_information_life_cycle_status_' + $ctrl.notebook.stateInfo.state | translate"
-                                                    ></span>
-                                                </span>
+                            <span
+                                data-ng-bind="'pci_notebooks_general_information_life_cycle_status_' + $ctrl.notebook.stateInfo.state | translate"
+                            ></span>
+                        </span>
                     </oui-tile-description>
 
-                    <oui-action-menu aria-label="Actions" data-compact data-placement="end">
-                        <oui-action-menu-item data-on-click="$ctrl.onNotebookStartClick($ctrl.notebook.id)"
-                                              data-disabled="!$ctrl.notebook.isStopped()">
+                    <oui-action-menu
+                        aria-label="Actions"
```

Comment:
Maybe translate the aria-label

Reply:
Not in the scope of this PR and I do not want to wait for a trad before merging this fix

Reply:
Ok my bad, I didn't notice it was already here